### PR TITLE
SmartyStreets.standardize uses zipcode

### PR DESCRIPTION
**Pull request by tmornini**

### CODE REVIEW

**Comment by tmornini** [comment]
@kballenegger Ping. :-)

File: lib/validate-address-smartystreets.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def self.verify(address)
         s.street   = x if x = address[:street] || address['street']
         s.city     = x if x = address[:city]   || address['city']
         s.state    = x if x = address[:state]  || address['state']
-        s.zip_code = x if x = address[:zip]    || address['zip']
+        s.zipcode = x if x = address[:zip]    || address['zip']
```
Comment (kballenegger) [inline lib/validate-address-smartystreets.rb:91]: Keep the alignment here.

**Comment by kballenegger** [comment]
@tmornini this looks good! Sorry I've been so slow to reply - I never notice things on GitHub where I'm not mentioned. The notification system could use some improvements! I'm merging this, (and I'll fix the missing space, so don't worry about it.)

**Comment by tmornini** [comment]
@kballenegger Sorry about the space, thank for fixing!

**Pull request merged**